src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/Touch/TouchBehavior.shared.cs

```diff
@@ -926,13 +926,13 @@ internal VisualElement? Element
 				gestureManager.Reset();
 				SetChildrenInputTransparent(false);
 			}
-			gestureManager.AbortAnimations(this, CancellationToken.None).SafeFireAndForget<TaskCanceledException>(ex => Trace.WriteLine(ex));
+			gestureManager.AbortAnimations(this, CancellationToken.None).SafeFireAndForget<TaskCanceledException>(ex => Trace.TraceError("{0}", ex));
```

Comment:
I think this should just be ex.Message to avoid spamming developers logs. A `TaskCanceledException` can be considered a valid scenario and therefore I don't think we need to log too much